Decide/src/main/java/DD2480/decide/CMVHandler.java

```diff
@@ -54,30 +54,11 @@ private boolean licOne() {
 		if(dataPoints.length<3){
 			return false;
 		}
-		//first 3 consecutive datapoints
-		Points first = dataPoints[0];
-		Points second = dataPoints[1];
-		Points third = dataPoints[2];
 		boolean keepSearching = (dataPoints.length>3) ? true : false;
```

Comment:
KeepSearching isn't used anywhere right?

Reply:
correct, mistake on my part this is legacy code, should be removed